src/components/tooltip/README.md

```diff
@@ -0,0 +1,69 @@
+# Tooltip
+
+#### Description
+
+Tooltips display informative text when users hover over or focus on an element.
+
+#### Usage
+
+```js
+<Tooltip
+  position="left"
+  label="If you buy a pizza, you will also get a free ice cream :)"
+>
+  <button onClick={orderPizza({ freeIceCream: 'yes please' })}>Submit</button>
+</Tooltip>
+```
+
+#### Working with disabled child elements
```

Comment:
We should mention that some of our components already support that (e.g. buttons, etc)